File: 0001_Two_Sum/solution.py
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        for i in range(len(nums)):
            for j in range(i+1, len(nums)):
                if nums[i]+nums[j]==target:
                    return [i,j]
        return []

if __name__=="__main__":
    sol = Solution()
    target=int(input("Enter target: "))
    nums = []
    print("Enter numbers, stop with non-numeric input:")
    while True:
        try:
            number= int(input())
            nums.append(number)
        except ValueError:
            break
    result = sol.twoSum(nums, target)
    if result:
        print("Solutions: [ {result[0]}, {result[1]}]")

File: 0001_Two_Sum/solution.cpp
#include <iostream>
#include <vector>
using namespace std;
class Solution { 
    public:
        vector<int> twoSum(vector<int>& nums, int target) {
            int lenght = nums.size();
            for(int i=0;i<lenght;i++){
                for(int j=i+1;j<lenght;j++){
                    if(nums[i]+nums[j]==target){
                        return {i,j};
                    }
                }
            }
            return {};
        }
    };
int main() {
Solution sol;
int target;
vector<int> nums;
cout << "Enter the target: ";
cin >> target;
cout << "Enter the numbers): ";
int number;
while(cin>>number){
    nums.push_back(number);
}
vector<int> result = sol.twoSum(nums, target);
cout<<"Solutions: ["<<result[0]<<", "<<result[1]<<"]"<<endl;  
    return 0;
}